lib/practice_exercises.rb

```diff
@@ -1,13 +1,24 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  new_array = []
+    list.each do  |i|   
+    unless new_array[-1] == i
+      new_array << i
+    end
+  end
+  return new_array 
 end
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(1)
 def longest_prefix(strings)
```

Comment:
That works, very clever because it's finding the max and min strings alphabetically.